video_search.py

```diff
@@ -0,0 +1,59 @@
+import json
+import pdb
+import re
+import urllib
+from BeautifulSoup import BeautifulSoup
+from mechanize import Browser
+
+start_link = "https://api.import.io/store/connector/_magic?url=https%3A%2F%2F"
```

Comment:
start_link and end_link are used only inside `create_link` function. They need not be global.